Attribute/AsDocumentListener.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Bundle\MongoDBBundle\Attribute;
+
+use Attribute;
+
+/**
+ * Service tag to autoconfigure document listeners.
+ */
+#[Attribute(Attribute::TARGET_CLASS | Attribute::IS_REPEATABLE)]
+class AsDocumentListener
+{
+    public function __construct(
+        public ?string $event = null,
+        public ?string $method = null,
+        public ?bool $lazy = null,
+        public ?string $connection = null,
```

Comment:
I'm not really sure if this is correct as this doesn't seem to be documented well. I was expecting passing `document_manager` name instead of connection here.

Reply:
Checked with https://github.com/symfony/doctrine-bridge/blob/5.3/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPass.php and it seems that it's correct.

Reply:
Documentation confirms that :) https://www.doctrine-project.org/projects/doctrine-mongodb-bundle/en/4.3/events.html#event-listeners